teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
<p>Question: In the explanation you write that missing atoms are added, in the signature missing residues. So just wondering if it adds missing atoms to an existing residue, or if it also adds/models complete missing residues.</p><p> </p><p>Actually I see part of the answer comes implicitly later in the function comments, maybe adapt the docstring slightly.</p><p> </p><p>Also I'm wondering what you mean by 2*?</p><blockquote>pdbfixer does a poor job in modeling missing residues</blockquote><p> </p><p>nonstandard, non standard or non-standard</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Added some explanations. I hope I was able to clear it up a bit</p>